### DISCUSSION TITLE
Incorrect license information on Savannah

### DISCUSSION THREAD

**Discussion by LiberalArtist** [libunwind/libunwind#628] [General]
I noticed that the Savannah project page for libunwind (<https://savannah.nongnu.org/projects/libunwind/>) lists libunwind as being under the X11 license.

AFAICT, libunwind is actually under the MIT (in SPDX terms—aka Expat) license. That's what's in the license files in this repository contain, and its also what the website (at <https://www.nongnu.org/libunwind/download.html>) and [the Debian copyright file](https://metadata.ftp-master.debian.org/changelogs//main/libu/libunwind/libunwind_1.3.2-2_copyright) say the license is.

Unfortunately, some other references seem to have used the metadata from Savannah: at least, the [Guix package of libunwind](https://guix.gnu.org/en/packages/libunwind-1.5.0/) currently says it is under the X11 license.

**Comment by bregma** [comment]
Moving to discussions because this is an issue with `nongnu.org` and not `github.com/libunwind`.
